code/controllers/subsystem/statpanel.dm

```diff
@@ -258,6 +254,20 @@ SUBSYSTEM_DEF(statpanels)
 	SIGNAL_HANDLER
 	SSstatpanels.cached_images -= REF(src)
 
+/// Stat panel window declaration
+/client/var/datum/tgui_window/stat_panel
+
+/**
+ * Handles incoming messages from the stat-panel TGUI.
+ */
+/client/proc/on_stat_panel_message(type, payload)
+	if(type == "Update-Verbs")
+		init_verbs()
+		return
+	if(type == "Remove-Tabs")
+		panel_tabs -= payload["tabs"]
+		return
```

Comment:
Currently unused, iirc the reason was:

> ahh
> ill probably need to make the queue into an associative list
> so i can make it use Byond.sendMessage instead of Byond.command
> to remove those verbs

From what I gathered, the `connected` flag is only false when not fully initialized, then it's set to true, and it never resets to false, so I feel like this whole command queuing construct is built for nothing.

``` js
// Any BYOND verb call must go through this, as if a verb is sent during reconnect then
// it will cause the reconnect to fail.
// This function will either call immediately, or queue until
// BYOND confirms we are connected.
function send_byond_command(command) {
```

So maybe this queuing function doesn't have to exist at all, which will free you up to use Byond.command() and Byond.sendMessage() directly.